Initial release

litvintech opened the pull request:
- Improved web3 dapp
- Implemented post Registry initialization with EntryCore
- Implemented divided logic for Entry creation/setting data
- Implemented versioning for Registry Creator
- Improved policies/security
- Implemented funding for Registries/Entries
- Added test coverage + report
- Added docs + contracts pipeline
- Refactoring

litvintech commented:
after review and merge going to set v0.1 tag to master

litvintech commented:
@xhipster @hleb-albau @Pechalka waiting for your review guys

hleb-albau commented on contracts/registry/RegistryPermissionControl.sol:
```diff
@@ -0,0 +1,85 @@
+        return admin;
+    }
+    
+    function getEntryPermissions()
```
fix getter name

hleb-albau commented on contracts/common/RegistrySafe.sol:
```diff
@@ -1,29 +1,60 @@
+* @dev Allows store etheirs and claim them by owner
+* @notice not recommend to use before release!
+*/
+contract RegistrySafe {
```
Maybe rename, not including `Registry` in name part? A bit confusing.

hleb-albau replied:
is Ownable?

hleb-albau commented on test/common/AdminableTests.js:
```diff
@@ -0,0 +1,70 @@
+
+const AdminableTestContract = artifacts.require('AdminableTestContract');
+
+contract('AdminableTestContract', function (accounts) {
```
Remove, no more `Adminable`

SaveTheAles commented on README.md:
```diff
@@ -1,9 +1,8 @@
 </h1>
 
-
 <h3 align="center">Most expensive Registry</h3>
```
Most expensive Registry isn't correct. I propose edit "most" to "the most"

SaveTheAles commented on README.md:
````diff
@@ -37,207 +39,60 @@ Entry type is defined during creation, so you can put into entry any custom logi
-```
+## [Configuring and deploying](https://cybersearch.io/Chaingear/development/)
 
+## General Chaingear/Registry ipeline
````
Edit ipline to pipeline

litvintech commented:
@SaveTheAles @xhipster @hleb-albau 
1. refactored all small issues, which you described above
2. added check for entry zero balance before deletion
3. added symbol to return values of registryInfo, refactored registryInfo method
4. added method which allows changing of registry name
5. fixed auth check for entry updates
6. added the example for a unique field to EntryCore
7. increased tests coverage
